python/lsst/pipe/tasks/multiBand.py

```diff
@@ -111,12 +116,117 @@ def setDefaults(self):
+across multiple filter bands. Two things must be achieved to finish coadd processing. Coadding individual 
+visits requires each exposure to be warped. This introduces covariance in the noise properties across pixels. 
+We correct the coadd variance by scaling the variance plane in the coadd to match the observed variance - this 
+is not the right thing to do but is simple and works well in practice. We must also detect sources and 
+generate footprints for the parent sources - this is delegated to the existing SourceDetectionTask.
+
+  \par Inputs:
+    deepCoadd{tract,patch,filter}: ExposureF
+  \par Outputs:
+    deepCoadd_det{tract,patch,filter}: SourceCatalog (only parent Footprints)
+    \n deepCoadd_calexp_det{tract,patch,filter}: ExposureF
+    \n deepCoadd_calexp_detBackground{tract,patch,filter}: BackgroundList
+  \par Data Unit:
+    tract, patch, filter
+
+DetectCoaddSourcesTask delegates most of its work to \ref SourceDetectionTask_ "SourceDetectionTask". You can 
+replace this subtask if you wish.
+
+\section pipe_tasks_multiBand_DetectCoaddSourcesTask_Initialize       Task initialization
+
+\copydoc \_\_init\_\_
+
+\section pipe_tasks_multiBand_DetectCoaddSourcesTask_Run       Invoking the Task
+
+\copydoc run
+
+\section pipe_tasks_multiBand_DetectCoaddSourcesTask_Config       Configuration parameters
+
+See \ref DetectCoaddSourcesConfig
+
+\section pipe_tasks_multiBand_DetectCoaddSourcesTask_Debug		Debug variables
+
+The \link lsst.pipe.base.cmdLineTask.CmdLineTask command line task\endlink interface supports a
+flag \c -d to import \b debug.py from your \c PYTHONPATH; see \ref baseDebug for more about \b debug.py files.
+
+DetectCoaddSourcesTask has no debug variables of its own because it relegates all the work to 
+\ref SourceDetectionTask_ "SourceDetectionTask"; see the documetation for 
+\ref SourceDetectionTask_ "SourceDetectionTask" for further information.
+
+\section pipe_tasks_multiband_DetectCoaddSourcesTask_Example	A complete example of using DetectCoaddSourcesTask
+
+DetectCoaddSourcesTask is meant to be run after assembling a coadded image in a given band. The purpose of 
+the task is to update the background, detect all sources in a single band and generate a set of parent 
+footprints. Subsequent tasks in the multi-band processing pipeline will merge sources across bands and, 
+eventually, perform forced photometry. Command-line usage of DetectCoaddSourcesTask expects a data reference 
+to the coadd to be processed. A list of the available optional arguments can be obtained by calling 
+detectCoaddSources.py with the --help command line argument:
+\code
+detectCoaddSources.py --help
+\endcode
+
+To demonstrate usage of the DetectCoaddSourcesTask in the larger context of multi-band processing, we will 
+process the HSC-I & -R band coadds generated from HSC engineering test data provided in the ci-hsc package. To 
```

Comment:
Please make sure you get the package name correct -- ``ci_hsc`` rather than ``ci-hsc`.